src/plugins/elasticsearch/lib/__tests__/health_check.js

```diff
@@ -53,7 +57,8 @@ describe('plugins/elasticsearch', function () {
       // Setup the server mock
       server = {
         log: sinon.stub(),
-        config: function () { return { get: get }; },
+        info: { port: 5601 },
+        config: function () { return { get: get, set: set }; },
```

Comment:
Don't bother blocking this PR on it, but in the future you can just do `{ get, set }`.